package.json

```diff
@@ -31,11 +31,20 @@
     "@types/node": "11.13.4",
     "@types/node-fetch": "2.3.2",
     "@types/time-span": "2.0.0",
+    "@typescript-eslint/eslint-plugin": "^4.3.0",
+    "@typescript-eslint/parser": "^4.3.0",
     "async-retry": "1.2.3",
     "conventional-changelog": "3.1.18",
     "conventional-changelog-cli": "2.0.35",
     "copyfiles": "2.3.0",
     "cz-conventional-changelog": "3.1.0",
+    "eslint": "^7.10.0",
+    "eslint-config-standard": "^14.1.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prefer-arrow": "^1.2.2",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
```

Comment:
Just sanity: are all these dependencies needed?

Reply:
Yes. I used the initial eslint file configurator. It already installs all the development dependencies necessary for the rules to work in typescript.